_careers/data.md

```diff
@@ -0,0 +1,140 @@
+
+- Data extraction: from structured sources and not structured ones (web pages, PDFs). We already have many data ingestion processes running, you'll help improve them if needed.
+- Quality control of data: we have started to build some automated checks to validate the quality of new data. You'll help consolidate and improve it
+- Data cleaning: you know the drill :)
+- Data analysis and exploratory visualizations to build, communicate and reason about hipothesis
+- Prediction and clasification: we have already built some initial models to predict the number of offers a tender will receive. We want to do many other things in this front. Predict prices, discounts, detect possible frauds...
+- Entity extraction: we have tons of PDFs with unstructured content (companies, concepts, contract clauses...). Once we have extracted them, many important products can be built.
+
+And many other challenges that will appear along the way. We'll start bit by bit and grow the team which you may end up managing (if that's something you'd want, that is).
+
+We use Jenkins to automate tasks. We are a Ruby On Rails shop, so some Ruby knowledge would be ideal, but we understand you'll be using Python and the usual suspects.
+
+You'll also be contributing from time to time to Gobierto Transparency, our open source solution for transparency and accountability initiatives.
+
+We work on [Github](http://github.com/populatetools/) using a git workflow with CI/CD, tests, different automations... Our workflow has been polished throughout a few years, so have automated many things (monitoring, error tracking, server setups & updates, deployments, various checks...). You can take a look at the [Gobierto repo to see how we actually work](http://github.com/populatetools/gobierto).
+
+
+## Day to day work
+
+- **Plan** new features of Gobierto and existing applications, and contribute to new apps from scratch
+- **Implement** defined features, manage your sprints, do sprint reviews
+- **Collaborate** with other team members: provide feedback and do code reviews
+- **Improve** and maintain existing code of production applications
+
+
+## Your first week
+
+(in no particular order)
+
+- Meet the team!
+- Set up your tools, install Gobierto
+- Browse our internal doc wiki to check out how we work, where to find things, etc.
+- Browse around in our general issue tracker (Github)
+- Get assigned initial issues to navigate around our development process: tests, deploy, code reviews...
+
+
+## Your first quarter
+
+- Master all of our internal tools and processes
+- You are tasked with planning a medium-sized feature in Gobierto that will take you several weeks to implement
+- Propose the work plan and work with a designer and a front-end engineer to implement it
+- Deploy code to staging, demo your work to the team/client, write documentation, polish it and merge it to master
```

Comment:
- You understand Gobierto contratos data model
- You are tasked with some small projects 
- A project has been released and is in production, adding value to the product

Reply:
ok